.idea/compiler.xml

```diff
@@ -21,14 +21,11 @@
         <module name="kroxylicious-schema-validation" />
```

Comment:
We should be dropping `.idea/compiler.xml` in the same way we are dropping the `iml` files right?